llama_stack/providers/utils/telemetry/tracing.py

```diff
@@ -259,10 +259,6 @@ def wrapper(*args, **kwargs):
         return wrapper
 
 
-def span(name: str, attributes: Dict[str, Any] = None):
```

Comment:
is there a reason you removed this shortened wrapper? `with tracing.span()` feels short and nice instead of `with tracing.SpanContextManager()`

Reply:
@dineshyv I don't think it was used as a decorator in the cases where you changed it. it was used as a context manager. this method is just a wrapper literally and I think it is useful as a shorthand.

Reply:
I wanted to explicitly prevent usage of the decorator. i can add it back.

Reply:
oh I see. but yes, let's add it back -- I'd rather see people writing `with tracing.span()` rather than `with tracing.SpanContextManager()`

Reply:
done.